bots/images/scripts/windows-10.bootstrap

```diff
@@ -0,0 +1,79 @@
+
+unzip -o MSEdge.Win10.VMWare.zip
+
+cat <<'EOF' > Autorun.inf
+[autorun]
+open=install.bat
+label=Selenium installer
+EOF
+
+cat <<'EOF' > selenium.bat
+powershell -Command "while (!(Get-NetIPAddress -AddressFamily ipv4 | Where-Object IPAddress -eq '10.111.112.10')) { sleep 2 }"
+timeout 10
+START "hub" java -jar C:\selenium\selenium.jar -role hub
+timeout 20
+java -Dwebdriver.edge.driver=c:\selenium\MicrosoftWebDriver.exe -jar C:\selenium\selenium.jar -role node  -hub http://localhost:4444/grid/register -browser browserName="MicrosoftEdge",platform=WINDOWS
+timeout 50
+EOF
+
+cat <<'EOF' > ethernet-config.bat
+net file 1>nul 2>nul && goto :run || powershell -ex unrestricted -Command "Start-Process -Verb RunAs -FilePath '%comspec%' -ArgumentList '/c %~fnx0 %*'"
+goto :eof
+:run
+netsh interface ipv4 set address name="Ethernet" static 10.111.112.10 255.255.240.0
+netsh interface ipv4 set address name="Ethernet 2" static 10.111.112.10 255.255.240.0
+netsh interface ipv4 set address name="Ethernet 3" static 10.111.112.10 255.255.240.0
+netsh interface ipv4 set address name="Ethernet 4" static 10.111.112.10 255.255.240.0
+EOF
+
+cat <<'EOF' > install.bat
+net file 1>nul 2>nul && goto :run || powershell -ex unrestricted -Command "Start-Process -Verb RunAs -FilePath '%comspec%' -ArgumentList '/c %~fnx0 %*'"
+goto :eof
+:run
+netsh advfirewall firewall add rule name="Open Port 4444" dir=in action=allow protocol=TCP localport=4444
+netsh advfirewall firewall add rule name="Open Port 5555" dir=in action=allow protocol=TCP localport=5555
+netsh advfirewall set allprofiles state off
+powershell -Command "$action=New-ScheduledTaskAction -Execute 'C:\selenium\ethernet-config.bat'; $trigger=New-ScheduledTaskTrigger -AtLogOn; Register-ScheduledTask -action $action -Trigger $trigger -TaskName 'Cockpit configure ethernet' -RunLevel 'Highest'"
+powershell -Command Mount-DiskImage -ImagePath "C:\virtio-win.iso"
+powershell -Command "(Get-DiskImage 'C:\virtio-win.iso' | Get-Volume).DriveLetter" > result.txt
+set /p VOLUME=<result.txt
+del result.txt
+pnputil.exe -i -a %VOLUME%:\qxldod\w10\amd64\qxldod.inf
+mkdir c:\selenium
+echo f | xcopy /f /y d:\ethernet-config.bat c:\selenium\ethernet-config.bat
+echo f | xcopy /f /y d:\selenium.jar c:\selenium\selenium.jar
+echo f | xcopy /f /y d:\selenium.bat c:\selenium\selenium.bat
+echo f | xcopy /f /y d:\MicrosoftWebDriver.exe c:\selenium\MicrosoftWebDriver.exe
+REG ADD "HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Run" /V "Selenium Hub" /t REG_SZ /F /D "C:\selenium\selenium.bat"
+d:\java-installer.exe /s
+timeout 30
+shutdown -s -t 0
+EOF
+
+CR=$(printf '\r')
+sed -i "s/\$/$CR/" *.bat Autorun.inf
+genisoimage -output wininit.iso -volid cidata -joliet *.bat *.exe selenium.jar Autorun.inf
+echo "Converting vmdk to qcow2..."
+qemu-img convert -f vmdk -O qcow2 disk.vmdk "$ORIGIN"
+virt-install -n "$VM_NAME" -r 4000 --vcpus=4 --os-type=windows --os-variant=win10 \
```

Comment:
4 CPUs is quite heavy - is that some sort of win10 minimum requirement, or would 2 be enough, too?

Reply:
Took those from the windows-8 bootstrap script, but we could reduce it I think. The memory might be harder.

Reply:
I see, nevermind then.